src/api/app/jobs/consistency_check.rb

```diff
@@ -58,6 +59,15 @@ def check_project(fix = nil)
       @errors << import_project_from_backend(ENV['project'])
       project = Project.get_by_name(ENV['project'])
     end
+    # check backend side
+    begin
+      Suse::Backend.get("/source/#{project.name}")
+    rescue ActiveXML::Transport::NotFoundError
+      Suse::Backend.without_global_write_through do
```

Comment:
okay, done in all places in request https://github.com/openSUSE/open-build-service/pull/2901